zerver/views/realm.py

```diff
@@ -273,6 +276,22 @@ def update_realm(
     # `REQ/has_request_variables` extraction.
     req_vars = {k: v for k, v in list(locals().items()) if k in realm.property_types}
 
+    for setting_name, setting_config in Realm.realm_permission_group_settings.items():
+        setting_id_name = setting_name + "_id"
+        if setting_id_name not in req_vars:  # nocoverage
+            continue
```

Comment:
Could we write this as just `assert setting_id_name in req_vars`, since we don't expect the case where that doesn't happen to be possible?